RileyLinkBLEKit/RileyLinkBLEDevice.m

```diff
@@ -426,10 +430,6 @@ - (void)peripheral:(CBPeripheral *)peripheral didUpdateNotificationStateForChara
     NSLog(@"Updated notification state for %@, %@", characteristic, error);
 }
 
-- (void)peripheralDidUpdateName:(CBPeripheral *)peripheral {
-    [[NSNotificationCenter defaultCenter] postNotificationName:RILEYLINK_EVENT_LIST_UPDATED object:nil];
```

Comment:
Still using it here: https://github.com/ps2/rileylink_ios/blob/master/RileyLinkKit/RileyLinkDeviceManager.swift#L37